DllImportGenerator/DllImportGenerator/Marshalling/SafeHandleMarshaller.cs

```diff
@@ -64,11 +66,8 @@ public IEnumerable<StatementSyntax> Generate(TypePositionInfo info, StubCodeCont
             switch (context.CurrentStage)
             {
                 case StubCodeContext.Stage.Setup:
-                    yield return LocalDeclarationStatement(
-                        VariableDeclaration(
-                            AsNativeType(info),
-                            SingletonSeparatedList(
-                                VariableDeclarator(nativeIdentifier))));
+                    yield return MarshallerHelpers.DeclareWithDefault(AsNativeType(info), nativeIdentifier);
```

Comment:
Since we’re already factoring this out, would it be possible to instead emit it in StubCodeGenerator when UsesNativeIdentifier returns true? That would make it easier to evolve the marshallers when we want to add support for emitting struct marshalling or Reverse P/Invoke marshalling.